### ISSUE TITLE
Testing Report a Problem Page Images for Accessibility

### ISSUE THREAD

**Issue by bpmcneilly** [internetarchive/openlibrary#4648]
# Page to Test:

Report a Problem - https://openlibrary.org/contact?path=/people/bpmcneilly

# Steps

Steps for this task are outlined in #4521

# Expectations

Test the specified page according to the steps outlined in the parent issue. Create individual bugs with steps to reproduce for any found violations on this page. Reference those bugs on this tracking issue.

Once the page has been tested & all bugs are found, close this issue.

**Comment by bpmcneilly** [comment]
@Sabreen-Parveen has tested & we are done!
